### ISSUE TITLE
Accessibility of clientside validation

### ISSUE THREAD

**Issue by mpruy** [in2code-de/powermail#1097]
The client side validation has a problem with accessibility.

An aria attribute (aria-live="polite") and the role (role="alert") should be added to the error messages so that screen readers can capture and reproduce the appearance of the messages.

Is it possible to add the necessary accessibility attributes?

**Comment by mschwemer** [comment]
We are aware of this issue and it will be fixed. It's a duplicate of #962

So I am closing it
